k8s.gcr.io/images/k8s-staging-kind/images.yaml

```diff
@@ -1 +0,0 @@
-# No images yet
```

Comment:
I'm working on this right now, unfortunate timing

Reply:
i can do that later Ben